README.md

```diff
@@ -23,36 +50,79 @@ that's been made default from that version forward.
 
 ```ruby
-gem 'webpacker', github: 'rails/webpacker'
+# Gemfile
+gem 'webpacker', github: '~> 1.2'
+```
+
+## Integrations
+
+Webpacker also ships with basic out-of-the-box integration for React, Angular, Vue and Elm and you can add them in your Rails app like so,
+
+```bash
+# For Rails 5.1+
+rails new webpacker-example-app --webpack=react
+rails new webpacker-example-app --webpack=angular
+rails new webpacker-example-app --webpack=vue
+```
+
+Or, by running installer manually once you have
+bundled the `webpacker` gem into your `Gemfile`
+
+```bash
+# Available integrations - [react, angular, vue or elm]
+rails webpacker:install:react
 ```
 
-You can also see a list of available commands by running `./bin/rails webpacker`
+You can see a list of available commands/tasks by running `./bin/rails webpacker`
 
 ## Binstubs
 
 Webpacker ships with two binstubs: `./bin/webpack` and `./bin/webpack-dev-server`.
-They're thin wrappers around the standard webpack.js executable, just to ensure that the right configuration file is loaded depending on your environment.
+They're thin wrappers around the standard webpack.js executable,
+just to ensure that the right configuration file is loaded depending on your environment.
 
-In development, you'll need to run `./bin/webpack-dev-server` in a separate terminal from `./bin/rails server` to have your `app/javascript/packs/*.js` files compiled as you make changes. If you'd rather not have to run the two processes separately by hand, you can use [Foreman](https://ddollar.github.io/foreman). `./bin/webpack-dev-server` launches the [Webpack Dev Server](https://webpack.js.org/configuration/dev-server/), which serves your pack files on http://localhost:8080/, and provides advanced Webpack features, such as [Hot Module Replacement](https://webpack.js.org/guides/hmr-react/).
+You can also pass cli options supported by [webpack-dev-server](https://webpack.js.org/configuration/dev-server/). Please note that inline
+options will always take precedence over the ones
+already set in the configuration file.
+
+```bash
+./bin/webpack-dev-server --host 0.0.0.0
+```
+
+In development, you'll need to run `./bin/webpack-dev-server` in a separate terminal from `./bin/rails server` to have your `app/javascript/packs/*.js` files compiled as you make changes.
+
+If you'd rather not have to run the two processes separately by hand, you can use [Foreman](https://ddollar.github.io/foreman). `./bin/webpack-dev-server` launches the [Webpack Dev Server](https://webpack.js.org/configuration/dev-server/), which serves your pack files on http://localhost:8080/, and provides advanced Webpack features, such as [Hot Module Replacement](https://webpack.js.org/guides/hmr-react/).
+
+We recommend using `webpack-dev-server` during development for better experience, however if you don't want that you can run `webpack` binstub in watch mode.
+
+```bash
+./bin/webpack --watch --progress
```

Comment:
Out of curiosity why does webpacker use 8080 by default when puma uses 3000?

I think it's worth noting in the README here that in order for your JS to work without running `webpack-dev-server` you need to set the port to match your puma port. For feature specs this also requires manually setting the Capybara puma port to something and then setting that same port in the `test` block of `development.server.yml`